nginx/tile_multiplier.py

```diff
@@ -3,15 +3,24 @@
 import sys
 
 min_zoom, max_zoom = [int(z) for z in sys.argv[1:3]]
+tile_set = set()
+
+def print_once(z, x, y):
+    if min_zoom <= z <= max_zoom:
+        tile = '{}/{}/{}'.format(z, x, y)
+        num_tiles = len(tile_set)
+        tile_set.add(tile)
+        if num_tiles < len(tile_set):
+            print tile
```

Comment:
```suggestion
        if tile not in tile_set:
            tile_set.add(tile)
            print(tile)
```

Reply:
I believe this the proposed change is less efficient. 

Both `not in` and `add` require the search for an existing `tile` in `tile_set`. 
The original code has one search operation in `add` and two `len` lookups,  which are probably much less expensive.

Reply:
ok, let keep as is.